libgnucash/engine/Account.cpp

```diff
@@ -5991,14 +5991,32 @@ gnc_account_delete_all_bayes_maps (Account *acc)
 /* ================================================================ */
 /* QofObject function implementation and registration */
 
+static void
+destroy_all_child_accounts (Account *acc, gpointer data)
```

Comment:
Just `destroy_account` or `destroy_account_cb`. It only destroys one.

Reply:
Not needed.